### PR TITLE
9975 BUG: Petitions "Served" QC Boxes Sorting in Reverse Order

### PR DESCRIPTION
**Pull request by rosiehsmith**
image
image

flexion#9975

### CODE REVIEW

**Review by mrinsin — APPROVED** [review]

File: web-client/src/presenter/computeds/formattedWorkQueue.ts [hunk 1]
```diff
@@ -387,20 +387,18 @@ export const formattedWorkQueue = (get, applicationContext) => {
...
-
-    sortField = [
+    highPriorityField = [
+      'highPriority',
+      'trialDate',
       workItemToSort => caseStatusSortRank[workItemToSort.caseStatus],
-      sortField,
     ];
-    sortDirection = ['asc', sortDirection];
+    highPriorityDirection = ['desc', 'asc', 'asc'];
```
Comment (ujahio) [inline web-client/src/presenter/computeds/formattedWorkQueue.ts:395]: Hey @rosiehsmith 
❓ : is `asc` purposefully repeated in the array? Thx

Comment (rosiehsmith) [reply web-client/src/presenter/computeds/formattedWorkQueue.ts:395]: good question! yes, it is. it corresponds to the `highPriorityField` array so the last two `asc` values are corresponding to `trialDate` and the `caseStatusSortRank` function

Comment (ujahio) [reply web-client/src/presenter/computeds/formattedWorkQueue.ts:395]: @rosiehsmith Thank you!

**Review by ujahio — APPROVED** [review]

**Comment by rosiehsmith** [comment]
I ran the 2 tests that [failed](https://github.com/ustaxcourt/ef-cms/actions/runs/4533313127/jobs/7986143704) from e2e cypress public locally and they were fine. This is just a GA runner issue.

**Pull request merged**